addons/l10n_no/data/account_tax_report_data.xml

```diff
@@ -14,16 +14,16 @@
     </record>
 
     <record id="account_tax_report_line_post_1" model="account.tax.report.line">
-        <field name="name">Post 1 Samlet omsetning og uttak utenfor merverdiavgiftsloven</field>
-        <field name="tag_name">Post 1 Samlet omsetning og uttak utenfor merverdiavgiftsloven</field>
+        <field name="name">Post 1</field>
+        <field name="tag_name">Post 1</field>
```

Comment:
@nla-odoo We only need to change the tag_name, not the name itself.

Reply:
@jco-odoo  Done